src/chat/chatter.py

```diff
@@ -387,6 +387,7 @@ def viewReplay(self):
     @QtCore.pyqtSlot()
     def viewVaultReplay(self):
         """ see the player replays in the vault """
+        self.lobby.client.replays.setCurrentIndex(2)  # focus on Online Fault
```

Comment:
but then `self.lobby.client.mainTabs.setCurrentIndex` should also be part of "get there", shouldn't it?